tests/wicked/before_test.pm

```diff
@@ -51,6 +46,20 @@ sub run {
         systemctl 'enable dhcpd.service';
         systemctl 'start dhcpd.service';
     }
+    if (check_var('WICKED', 'basic')) {
+        $self->get_from_data('wicked/check_interfaces.sh', '/data/check_interfaces.sh', executable => 1);
+        assert_script_run("rcwickedd restart");
+        if (check_var('IS_WICKED_REF', '0')) {
```

Comment:
Don't know if we set IS_WICKED_REF also  for SUT  always. Maybe `! check_var('IS_WICKED_REF', '1')` is more safe.

Reply:
we setup IS_WICKED_REF=0 in all the SUT tests, so we are safe I think :)